### PR TITLE
Closes #898 Update close icon on upsell banners and admin bar

### LINKED ISSUE #898: Update upsell banners and notification
The `x` - close icon on the pink upsell banners (settings and bulk optimization page) and notifications in the toolbar menu should be updated with the icon we are using in other banners and alignment should be improved according to our brand guidelines.

Screenshot 2024-09-11 at 12 22 49 (2)
Screenshot 2024-09-11 at 12 22 32
Screenshot 2024-09-11 at 12 21 55 (3)

Example with the icon we would like to use:
image

Mocflow docs available [here](https://wireframepro.mockflow.com/editor.jsp?editor=off&publicid=Me1f1d8daaf35fcf22a1b821e0be46cb11727077568136&projectid=M8CjSMDSGh&perm=Owner&space=default#/page/Debb1a169faa4947fc3a0c4eb0f2fe1af/mode/design).

NB: Please follow our brand guidelines for colors, icons, alignment, fonts, etc., as the Mocflow file is not a design file and may not always include every detail.

### PR DESCRIPTION
**Pull request by remyperona**
# Description

Fixes #898 

Update close icon to use default icon from WP

## Type of change

- [x] Enhancement (non-breaking change which improves an existing functionality).

## Detailed scenario

- With quota < 20%
- Display upsell banner on settings or bulk optimization page
- Display admin bar Imagify sub-menu
- Close icon is updated to default icon used in WP and in white

admin-bar
bulk
settings

### Documentation

Updated class used + CSS color

# Mandatory Checklist

## Code validation

- [x] I validated all the Acceptance Criteria. If possible, provide screenshots or videos.
- [x] I triggered all changed lines of code at least once without new errors/warnings/notices.

## Code style
- [x] I wrote a self-explanatory code about what it does.
- [x] I did not introduce unnecessary complexity.

### CODE REVIEW

**Review by wordpressfan — APPROVED** [review]

File: views/admin/admin-bar-status.php [hunk 1]
```diff
@@ -35,7 +35,7 @@
 		<?php endif; ?>
 		<p><?php echo $data['text']; ?></p>
 		<p class="center txt-center text-center"><a class="imagify-upsell-admin-bar-button" href="<?php echo esc_url( $data['upgrade_link'] ); ?>" target="_blank"><?php echo $data['button_text']; ?></a></p>
-		<a href="<?php echo esc_url( get_imagify_admin_url( 'dismiss-notice', 'upsell-admin-bar' ) ); ?>" class="imagify-notice-dismiss imagify-upsell-dismiss" title="<?php esc_attr_e( 'Dismiss this notice', 'imagify' ); ?>"><span class="screen-reader-text"><?php esc_html_e( 'Dismiss this notice', 'imagify' ); ?></span></a>
+		<a href="<?php echo esc_url( get_imagify_admin_url( 'dismiss-notice', 'upsell-admin-bar' ) ); ?>" class="imagify-notice-dismiss imagify-upsell-dismiss notice-dismiss" title="<?php esc_attr_e( 'Dismiss this notice', 'imagify' ); ?>"><span class="screen-reader-text"><?php esc_html_e( 'Dismiss this notice', 'imagify' ); ?></span></a>
```
Comment (Mai-Saad) [inline views/admin/admin-bar-status.php:38]: @Tabrisrp Is there a reason that we didn't remove `imagify-upsell-dismiss` here while we removed it in part-upsell.php file?

**Comment by Mai-Saad** [comment]
@Tabrisrp Thanks for the PR.
1- settings page :heavy_check_mark: 
settingsPage2
2- Bulk page => not sure why icon is grey + the one in the page banner has underline
inBulkOPtimization
3- admin bar from WP dashboard page
AnyWPAdminPage
4- the notice when click bulk optimization link while having no credit (not mentioned in main ticket but just here for reference)
WhenClickBulkOPtimizationLink

@Tabrisrp Any idea why 2,3 has the icon grey and may be underlined?

**Comment by remyperona** [comment]
I pushed an update for the CSS for cases 2 and 3.

**Comment by Mai-Saad** [comment]
Thanks, @Tabrisrp Looks good now.
Screenshot from 2024-10-04 12-15-34
Screenshot from 2024-10-04 12-15-45
Screenshot from 2024-10-04 12-15-23

**Pull request merged**